profile/README.md

```diff
@@ -2,10 +2,30 @@
 
 # Welcome!
 
-These projects will make it easy to interact with the Pinata APIs.
+We're glad you're here 🎉.
+
+In this space, you'll find tools to supercharge your Pinata developer experience.
+For detailed information, tutorials and plug-and-play code blocks, head over to our [documentation](https://docs.pinata.cloud/).
+
+## Tools
 
 ### [Pinata SDK](https://github.com/PinataCloud/Pinata-SDK)
-The Pinata NodeJS SDK provides the quickest / easiest path for interacting with the Pinata API.
+The official Pinata NodeJS SDK provides the quickest / easiest path for interacting with the Pinata API. For SDKs supporting other languages, head over to [this page](https://docs.pinata.cloud/sdks)
+
+### [Submarine SDK](https://github.com/PinataCloud/pinata-submarine)
+This SDK is designed to make working with the Submarine API to take advantage of keeping IPFS files, private. This toolkit makes Submarine so simple that any developer can build their own submarine.me.
+
+## Links
+
+**Join our [community](https://discord.com/invite/pinata)**
+
+
+**Follow us on socials**
+
+[Twitter](https://twitter.com/pinatacloud)
```

Comment:
could we maybe do:

[Twitter](https://twitter.com/pinatacloud)  |  [LinkedIn](https://www.linkedin.com/company/pinatacloud)  |  [YouTube](https://www.youtube.com/c/Pinatacloud)  |  [Instagram](https://www.instagram.com/pinata.cloud/)